nixos/tests/installed-tests/graphene.nix

```diff
@@ -2,4 +2,8 @@
 
 makeInstalledTest {
   tested = pkgs.graphene;
+
+  testConfig = {
+    environment.systemPackages = with pkgs; [ python3 ];
```

Comment:
Turns out the test does not actually work since pygobject is missing too: https://github.com/ebassi/graphene/blob/a28aacb00a103ac616374c025406f629ed6a5289/tests/introspection.py#L33

Reply:
I can always use the same manual patching as https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/graphene/default.nix#L64. That's probably cleaner than adding the dependency in the test.

Reply:
Oh I missed that skip test. Good catch :)